source/content/plugins-known-issues.md

```diff
@@ -523,7 +523,39 @@ ___
 
 ___
 
-## ManageWP Worker
+## Jetpack
+
+<ReviewDate date="2022-03-09" />
+
+**Issue:** [Jetpack](https://wordpress.org/plugins/jetpack/) requires the XMLRPC interface to communicate with Automattic servers. The Pantheon WordPress upstream [disables access to the XMLRPC endpoint](/wordpress-best-practices#avoid-xml-rpc-attacks) by default as it is a common scanning target for bots and sees lots of invalid traffic.
+
+**Solution:** Modify your site's `pantheon.yml` file to [allow access](/pantheon-yml#protected-web-paths-override) to the `xmlrpc.php` path:
+
+```yml:title=pantheon.yml
+protected_web_paths_override: true
+protected_web_paths:
+  - /private
+  - /wp-content/uploads/private
+```
+
+This will maintain the normal security settings for other paths, but allows access for XMLRPC.
+
+<Alert title="Note"  type="info" >
+
+Pantheon does not support XML-RPC if it is enabled.
```

Comment:
While we most certainly do not support this, we should probably include more documentation about how to secure a site that wants to utilize Jetpack from XMLRPC.php attacks.

If you disable protection by removing xmlrpc.php from protected paths, you should still try to block all xmlrpc.php requests EXCEPT from allowed IPs.

https://jetpack.com/support/how-to-add-jetpack-ips-allowlist/ 

You can use the list of IPS above, and within our [advanced redirects section for WordPress](https://pantheon.io/docs/advanced-redirects#restrict-access-to-paths-based-on-ip) add those ips to the `is_from_trusted_ip` function under the `trusted_ips` array. Then, modify the `disallow_uri` array to the following...

```
    $disallow_uri = array(
        '/wp-login.php',
        '/wp-admin/',
        '/xmlrpc.php',
    ); 

```

This would need to be tested, but I think it's the more responsible approach. Kudos to Kris.

Also a note on the [Protection Module to prevent brute-force attacks](https://jetpack.com/support/protect/#:~:text=Jetpack's%20Protect%20module%20collects%20information,B%2C%20it%20would%20be%20blocked.).  This will face similar hurdles as security plugins, and under enough traffic could harm the site or the file system itself. It's not a good idea to enable this, will likely earn itself a place under "Plugins with known issues", even though this is a module within a plugin.